docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -426,17 +453,94 @@
             - "Pod log: {{get_logs_pre_unlock.stdout}}"
             - "err: {{get_logs_pre_unlock.stderr}}"
 
-        # If the task "Wait until unlock task triggered" failed, we export some
-        # useful information to the fail playbook msg.
-        - name: Output dm logs
+        # if unlock was triggered, wait until unlock complete
+        # and check resources and logs.
+        - block:
+          - name: Waiting for port to become open
+            local_action:
+              module: wait_for
+                port={{ ansible_port }}
+                host={{ ansible_host }}
+                delay={{ boot_wait_time }}
+                timeout={{ wait_for_timeout }}
+                state=started
+            register: waiting_unlock
+            retries: 40
+            delay: 20
+            ignore_errors: yes
+
+          # Waiting task after unlock to catch right status
+          - wait_for:
+              timeout: 600
+              msg: Waiting after unlock due to apply DM config
+
+          # Get pod name to retrieve the logs
+          - name: Get dm pod name
```

Comment:
Please ensure to use a consistent naming style throughout the document. I would suggest to write "Deployment Manager" and "DM" (both in upper case).

Reply:
Done.